apps/homescreen/bower_components/base/.bower.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "base",
+  "authors": [
+    "Wilson Page <wilsonpage@me.com>"
+  ],
+  "main": "base.css",
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "homepage": "https://github.com/gaia-components/base",
+  "_release": "737e000325",
+  "_resolution": {
+    "type": "branch",
+    "branch": "master",
+    "commit": "737e0003253ee4f3ceff5c1223543a1b3f7d39dd"
+  },
+  "_source": "git://github.com/gaia-components/base.git",
+  "_target": "master",
+  "_originalSource": "gaia-components/base"
+}
```

Comment:
This is an external component, so I can't change it (see https://github.com/gaia-components/base)